site/en/docs/privacy-sandbox/fledge/index.md

```diff
@@ -561,26 +559,28 @@ A reporting mechanism for losing bidders is [under discussion](https://github.co
 {% endDetails %}
 
 
-{: #trusted-server-detail}
+{: #key-value-service-detail}
 
 {% Details %}
 
 {% DetailsSummary %}
-## What is a trusted server?
+## What is a FLEDGE Key/Value service?
 
 {% endDetailsSummary %}
 
-In the context of the Privacy Sandbox, a trusted server is a secure environment to enable access to
-data, while preserving privacy. A request to a trusted server cannot result in event-level logging,
-or have other side effects.
+FLEDGE Key/Value service allows adtechs to query for realtime data when a bid is made by the buyer or when an ad is scored by the seller while preserving privacy. FLEDGE Key/Value service is one of the [FLEDGE services](/blog/fledge-service-overview/). 
```

Comment:
Maybe 'when a bid is made by the buyer, and for sellers to score ads while preserving privacy'.

Reply:
updated